gitlab/resource_gitlab_project_access_token.go

```diff
@@ -0,0 +1,199 @@
+package gitlab
+
+import (
+	"fmt"
+	"log"
+	"strconv"
+	"time"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/validation"
+	gitlab "github.com/xanzy/go-gitlab"
+)
+
+func resourceGitlabProjectAccessToken() *schema.Resource {
+	// lintignore: XR002 // TODO: Resolve this tfproviderlint issue
+	return &schema.Resource{
+		Create: resourceGitlabProjectAccessTokenCreate,
```

Comment:
Please use the context-aware CRUD functions, see the [upgrade guide](https://www.terraform.io/plugin/sdkv2/guides/v2-upgrade-guide#more-support-for-context-context).

Also make sure to use the new `ctx` to pass it to the `gitlab` API functions as `gitlab.WithContext(ctx)`. It's usually the last parameter of the API functions.

Reply:
Resolved in last commit